src/core/listener.h

```diff
@@ -70,10 +102,15 @@ typedef struct QUIC_LISTENER {
 #endif
 
     //
-    // Active reference count on the listener.
+    // Active reference count on the listener preventing a stop completion.
     //
     CXPLAT_REF_COUNT RefCount;
 
+    //
+    // Internal reference count on the listener preventing cleanup.
+    //
+    CXPLAT_REF_COUNT InternalRefCount;
```

Comment:
Two separate ref counts seem like a design that invites bugs difficult bugs if they get out of sync. Is there a way to combine them, like how connection.h has `RefTypeCount` (for debugging purposes)?